script.js

```diff
@@ -33,6 +34,11 @@ const cartoSheetSyncTable =
 const cartoCountiesURI = createCountiesCartoURI();
 const cartoStatesURI = createStatesCartoURI();
 
+// colorScale comes from this ColorBrewer url:
+// https://colorbrewer2.org/#type=sequential&scheme=YlGn&n=7
+const fillColorScale = ["#808080","#d9f0a3","#78c679","#238443"];
```

Comment:
Maybe add a variable for the grey color and call it `colorNoData` or `colorNoScore` so that it can be referenced in a single place. This will make it easier to change later if desired.

Reply:
@jimboweb why not use the `colorNoData` var in the first index of the color arrays? It will reduce the room for error if the value of `colorNoData` changes in the future.